check.go

```diff
@@ -146,3 +178,24 @@ func getVersionMinor(ver string) string {
 	}
 	return fmt.Sprintf("%s.%s", comps[0], comps[1])
 }
+
+type SchemaHttpLoader http.Client
+
+func (l *SchemaHttpLoader) Load(url string) (interface{}, error) {
+	client := (*http.Client)(l)
+	resp, err := client.Get(url)
+	if err != nil {
+		return nil, err
+	}
+	if resp.StatusCode != http.StatusOK {
+		_ = resp.Body.Close()
+		return nil, fmt.Errorf("%s returned status code %d", url, resp.StatusCode)
+	}
+
+	body, err := jsonschema.UnmarshalJSON(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	return body, resp.Body.Close()
```

Comment:
I am not sure about this, but either we add a `resp.Body.Close()` before line 197, or we do not need to call `resp.Body.Close()` in line 200, no ? Does `jsonschema.UnmarshalJSON` call `Close` itself?

Reply:
`UnmarshalJSON` does not close the reader, but it's true that we should close it if there's an error